maestro-android/src/main/java/dev/mobile/maestro/handlers/LocaleSettingHandler.kt

```diff
@@ -0,0 +1,74 @@
+package dev.mobile.maestro.handlers
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.Configuration
+import android.os.Build
+import org.lsposed.hiddenapibypass.HiddenApiBypass
+import java.lang.reflect.InvocationTargetException
+import java.util.*
+
+class LocaleSettingHandler(context: Context) : AbstractSettingHandler(context, listOf(CHANGE_CONFIGURATION)) {
+    fun setLocale(locale: Locale) {
+        if (hasPermissions()) {
```

Comment:
Are we guaranteed permission always?

Reply:
it should be there always from what I've tested, for now it just checks that `CHANGE_CONFIGURATION` is there